cp-url.go

```diff
@@ -31,6 +31,15 @@ type copyURLs struct {
 	Error         *probe.Error `json:"-"`
 }
 
+func (m *copyURLs) Equal(o interface{}) bool {
```

Comment:
> The interface is being implemented is Comparer (see queue.go)

In such a case i think copyURLs and mirrorURLs can be of same type, i would generally avoid taking interfaces like this.

```
type mirrorURLs struct {
        SourceAlias   string
        SourceContent *clientContent
        TargetAlias   string
        TargetContent *clientContent
        Error         *probe.Error `json:"-"`
}

type copyURLs struct {
        SourceAlias   string
        SourceContent *clientContent
        TargetAlias   string
        TargetContent *clientContent
        Error         *probe.Error `json:"-"`
}

```
Both structs are same here.